sklearn/manifold/spectral_embedding_.py

```diff
@@ -263,7 +263,9 @@ def spectral_embedding(adjacency, n_components=8, eigen_solver=None,
         # problem.
         if not sparse.issparse(laplacian):
             warnings.warn("AMG works better for sparse matrices")
-        laplacian = laplacian.astype(np.float)  # lobpcg needs native floats
+        # lobpcg needs native floats
```

Comment:
I think in almost platforms we support `.astype(np.float)` is actually doing the same as `.astype(np.float64)`. It's just that I prefer not having any ambiguity when reading the code. 

I think the comment should be "lobpcg needs double precision floats".